ddtrace/internal/flare/flare.py

```diff
@@ -161,20 +172,20 @@ def _generate_payload(self, params: Dict[str, str]) -> Tuple[dict, bytes]:
             encoded_key = key.encode()
             encoded_value = value.encode()
             body.write(b"--" + boundary + newline)
-            body.write(b'Content-Disposition: form-data; name="{%s}"{%s}{%s}' % (encoded_key, newline, newline))
-            body.write(b"{%s}{%s}" % (encoded_value, newline))
+            body.write(b'Content-Disposition: form-data; name="%s"%s%s' % (encoded_key, newline, newline))
```

Comment:
The `{}` brackets were typos from the previous way that the string substitution was happening, and this was breaking the header format.